### ISSUE TITLE
4.1.6 - highcharts-3d.js breaks 3d Bar charts

### ISSUE THREAD

**Issue by hanoncs** [highslide-software/highcharts.com#4280]
The 4.1.6 update breaks 3d bar charts. From what I can tell the 4.1.6 highcharts.js if fine, I think it is the 4.1.6 highcharts-3d.js file.

I am using this as a base demo: http://jsfiddle.net/pktpkwzo/

Here is the broken 3d chart using 4.1.6: http://jsfiddle.net/pktpkwzo/2/

Here is a working chart with the same code, with 4.1.5 scripts: http://jsfiddle.net/pktpkwzo/3/

**Comment by highslide-software** [comment]
We disabled 3D for inverted charts because of a bug with the data label positioning, but it looks like we threw the baby out with the bath water. It seems to work well except the labels.

**Comment by highslide-software** [comment]
Here's a drop-in fix for v4.1.6:

```js
    Highcharts.Chart.prototype.is3d = function () {
	    return this.options.chart.options3d && this.options.chart.options3d.enabled; // #4280
    };
```
[View live on jsFiddle](http://jsfiddle.net/highcharts/pktpkwzo/4/)
